test/index.js

```diff
@@ -1,14 +1,15 @@
 var isBrowser = !(typeof module !== 'undefined' && module.exports);
-if (!isBrowser){
+if (!isBrowser) {
   // non-browser code
   var chai = require('chai');
   chai.should();
   var urlgrey = require('../index');
 }
+var assert = chai.assert;
```

Comment:
There aren't actual changes to the test file here -- we just need to use `assert` in a few spots rather than the `object.should.whatever` pattern we were using previously